pages/cloudflare/index.mdx

```diff
@@ -1,2 +1,54 @@
-### TODO
+import { SITE } from '../../config';
+import { Callout } from 'nextra/components';
 
+## Cloudflare
+
+---
+
+The [@opennextjs/cloudflare](https://www.npmjs.com/package/@opennextjs/cloudflare) adapter lets you deploy Next.js apps to [Cloudflare Workers](https://developers.cloudflare.com/workers) and [Cloudflare Pages](https://developers.cloudflare.com/pages).
+
+---
+
+<Callout>
+[@opennextjs/cloudflare](https://www.npmjs.com/package/@opennextjs/cloudflare) is pre 1.0, and still in active development. You should try it, [report bugs](https://github.com/opennextjs/opennextjs-cloudflare/issues), and [share feedback](https://github.com/opennextjs/opennextjs-cloudflare/discussions) to help make running Next.js apps on Cloudflare easier. We don't quite yet recommend using it for mission-critical production apps.
+
+You can also use [@cloudflare/next-on-pages](https://www.npmjs.com/package/@cloudflare/next-on-pages) to deploy Next.js apps to Cloudflare Pages. You can review the differences in supported Next.js features between @opennextjs/cloudflare and @cloudflare/next-on-pages [here](LINK).
+</Callout>
+
+### Get Started
+
+##### New apps
+
+To create a new Next.js app, pre-configured to run on Cloudflare using @opennextjs/cloudflare, run:
+
+```
+<command>
+```
+
+##### Existing Next.js apps
+
+Follow the guide [here](/cloudflare/get-started) to use [@opennextjs/cloudflare](https://www.npmjs.com/package/@opennextjs/cloudflare) with an existing Next.js app.
+
+### Supported Next.js features
+
+TODO: Check these, what else should / shouldn't be on here
+
+- [x] App & Pages Router
```

Comment:
I think the status is:

```
- [x] App & Pages Router
- [x] API routes
- [x] Dynamic routes
- [x] Static site generation (SSG)
- [x] Server-side rendering (SSR)
- [x] Incremental static regeneration (ISR)
- [ ] Partial Prerendering (PPR)
- [ ] Middleware
- [ ] Image optimization
- [ ] [NextAuth.js](https://next-auth.js.org)
- [ ] Experimental streaming support
```

At least that's what we have tested

Reply:
thank you!!!!

Reply:
` [ ] [NextAuth.js](https://next-auth.js.org)`
Should we focus on https://authjs.dev/ instead? v5 was rewritten to run better on edge. It's still in beta so it might be a little awkward to teeter on v4 and v5

Reply:
Actually going to omit since these are libraries and (for now) I'd like to keep this focused on things built into Next.js itself. Had copied this over from existing AWS list.

We can come back and do an actual list of supported/unsupported Next.js libraries. Good callout.